examples/cpp/vrs/README.md

```diff
@@ -5,17 +5,78 @@ tags = ["2D", "3D", "vrs", "viewer", "C++"]
 thumbnail = "https://static.rerun.io/vrs-viewer/28da92ebc2f0bccd5cf904314d2f8b0b0c45c879/480w.png"
 thumbnail_dimensions = [480, 480]
 -->
+[//]: # (> VRS is a file format optimized to record & playback streams of sensor data, such as images, audio samples, and any other discrete sensors &#40;IMU, temperature, etc&#41;, stored in per-device streams of time-stamped records.)
 
-
-This is an example that shows how to use [Rerun](https://github.com/rerun-io/rerun)'s C++ API to log and view [VRS](https://github.com/facebookresearch/vrs) files.
-
-> VRS is a file format optimized to record & playback streams of sensor data, such as images, audio samples, and any other discrete sensors (IMU, temperature, etc), stored in per-device streams of time-stamped records.
-
-You can find the example at <https://github.com/rerun-io/cpp-example-vrs>.
+[//]: # (You can find the example at <https://github.com/rerun-io/cpp-example-vrs>.)
```

Comment:
What syntax is this?

`[//]: #` is news to me. I can't find anything about it on https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

And what's with the `&#40;`?

Reply:
They don't show up in https://github.com/rerun-io/rerun/blob/andreasnaoum-examples-batch3/examples/cpp/vrs/README.md so are they comments, or what?

Reply:
Yes, they are comments. I will finalize it now with spell check, lint and also clear the comments

Reply:
`<!-- … -->` is the more common markdown comment style, from HTML

Reply:
I used the keyboard shortcut Cmd + "/" for commenting, and that happened. It was also weird for me tbh